.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
```

Comment:
Could you please add the name of the test cases already in place as a checklist?

Reply:
Try going through the testing demo video in the readme for the list of unit tests present.

Reply:
I went through the Demo video, and I already have asked to run the `python3 app.py -v`  command. So clearly I'm either missing out on something or have misinterpreted this. So can you elaborate on the issue?

Reply:
On running the unit tests, 5 different unit tests(testResults, testResultsJSON etc.) are run on the current state of the application.
Having a checklist for those 5 test cases is better.